ansible/roles/host_setup/vars/ubuntu.yml

```diff
@@ -1,7 +1,9 @@
 ---
 
 ## Defined required kernel
-host_required_kernel: 6.8.0-0-generic
+# This variable is used to set the default kernel in grub.  Ensure you are 
+# using the ENTIRE output from uname -r
+host_required_kernel: 6.8.0-47-generic
```

Comment:
The latest kernel in this series is `6.8.0-56-generic` if we're going down the path of doing specific kernel pinning, I don't think we should start out with an older version of the kernel that contains known CVEs.